modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/higher_level_reviews_controller.rb

```diff
@@ -25,10 +25,11 @@ class AppealsApi::V2::DecisionReviews::HigherLevelReviewsController < AppealsApi
   )['definitions']['hlrCreateParameters']['properties'].keys
   SCHEMA_ERROR_TYPE = Common::Exceptions::DetailedSchemaErrors
   ALLOWED_COLUMNS = %i[id status code detail created_at updated_at].freeze
+  ICN_REGEX = /^[0-9]{10}V[0-9]{6}$/.freeze
 
   def index
     veteran_hlrs = AppealsApi::HigherLevelReview.select(ALLOWED_COLUMNS)
-                                                .where(veteran_icn: request_headers['X-VA-ICN'].presence&.strip)
+                                                .where(veteran_icn: request_headers['X-VA-ICN'])
```

Comment:
Not needed anymore now that we're validating the ICN and returning a 422 in the `before_action` if it's not exactly matching the ICN format. 🙂